File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.kt
class Solution {
        fun solution(num_list: IntArray): Int {
            var answer: Int = 0
            var odd = ""
            var even = ""
            for (i in num_list.indices) {
                if (num_list[i] % 2 != 0) {
                    odd += num_list[i]
                }
            }
            for (i in num_list.indices) {
                if (num_list[i] % 2 == 0) {
                    even += num_list[i]
                }
            }
            answer = odd.toInt() + even.toInt()
            
            return answer
        }
    }

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer = 0
    odd = "".join(str(n) for n in num_list if n % 2 != 0)
    even = "".join(str(n) for n in num_list if n % 2 == 0)
    answer = int(odd) + int(even)
    return answer